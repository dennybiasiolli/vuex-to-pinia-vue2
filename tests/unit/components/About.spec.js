import { shallowMount } from '@vue/test-utils';
import About from '@/components/About.vue'

describe('About.vue', () => {
  test('should respect the snapshot', () => {
    const wrapper = shallowMount(About)
    expect(wrapper.element).toMatchSnapshot()
  })
})
