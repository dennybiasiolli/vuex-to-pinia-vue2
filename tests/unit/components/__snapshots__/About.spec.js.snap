// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue should respect the snapshot 1`] = `
<div>
  
  This is the About page

</div>
`;
